Formula/msc-generator.rb

```diff
@@ -34,8 +34,6 @@ class MscGenerator < Formula
 
   def install
     system "./configure", "--prefix=#{prefix}", "--disable-font-checks"
-    # Dance around upstream trying to build everything in doc/ which we don't do for now
-    # system "make", "install"
     system "make", "-C", "src", "install"
     system "make", "-C", "doc", "msc-gen.1"
```

Comment:
can we do this here then?
```suggestion
    system "make"
    system "make", "install"
```

Reply:
Can we use simply `make install`? (And remove `make` from your suggestion?)

Reply:
If that works, sure

Reply:
We guessed it would work, but it didn't. I reverted this change.